aspnetcore/grpc/authn-and-authz.md

```diff
@@ -103,70 +102,76 @@ private static GrpcChannel CreateAuthenticatedChannel(string address)
 
 #### Bearer token with gRPC client factory
 
-gRPC client factory can create clients that send a bearer token using `ChannelCredentials`. When configuring a client, assign the `CallCredentials` the client should use with the `ConfigureChannel` method.
+gRPC client factory can create clients that send a bearer token using `AddCallCredentials`. The delegate passed to `AddCallCredentials` is executed for each gRPC call.
```

Comment:
A colon is typical, but not critical, in these cases where the sentence points directly at a code example ...

```suggestion
gRPC client factory can create clients that send a bearer token using `AddCallCredentials`. The delegate passed to `AddCallCredentials` is executed for each gRPC call:
```

... another option is to make it say ...

> ... is executed for each gRPC call, as the following example demonstrates: